test/e2e/data/expected.yaml

```diff
@@ -0,0 +1,47 @@
+segmentItems:
```

Comment:
Correct segment data. Once the test passed, then basically C++ internal service works.

Reply:
The segment data looks good to me, just want to confirm that you didn't have a `peer` field for now, right?

Reply:
added `peer`